src/commands/captureElementScreenshot.js

```diff
@@ -1,39 +1,67 @@
-import generatePath from '../path-generator';
+import generatePaths from '../path-generator';
 import { hasProperty } from '../helpers';
 
 const { EventEmitter } = require('events');
 const Jimp = require('jimp');
 const fs = require('fs-extra');
-const path = require('path');
 
 module.exports = class CaptureElementScreenshot extends EventEmitter {
-    command(selector = 'body', filename = selector) {
+    command(selector = 'body', filename = selector, callback = () => {}) {
         const { api } = this.client;
         const { regression: settings } = api.globals.deviance;
+        const { name: testName, module: testModule } = api.currentTest;
+        const jimpOperations = [];
+        const filenames = generatePaths(settings, filename, testName, testModule);
 
-        process.nextTick(() => {
-            api.getLocation(selector, ({ value: { x, y } }) => {
-                api.getElementSize(selector, ({ value: { width, height } }) => {
-                    api.screenshot(false, (screenshotEncoded) => {
-                        Jimp.read(Buffer.from(screenshotEncoded.value, 'base64'))
-                            .then((image) => {
-                                const targetFilename = generatePath(settings, filename);
+        api.getLocation(selector, ({ value: { x, y } }) => {
+            api.getElementSize(selector, ({ value: { width, height } }) => {
+                api.screenshot(false, (screenshotEncoded) => {
+                    const results = {};
+                    jimpOperations.push(Jimp.read(Buffer.from(screenshotEncoded.value, 'base64')));
```

Comment:
I'd define `jimpOperations` here: `const jimpOpertations = [...]`

Reply:
Will make it simpler to refactor later if everything it defined at first usage

Reply:
Resolved